src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -4,8 +4,24 @@ import './UsernameEntry.css';
 import ListSelection from '../listSelection/ListSelection';
 
 class UsernameEntry extends React.Component {
-  sendReminder = async user => {
-    const result = await this.props.sendSlackMessage(user.id);
+  state = {
+    selectedUser: null
+  };
+
+  promptToConfirm = user => {
+    this.setState({selectedUser: user});
+  };
+
+  deselect = () => {
+    if (this.state.selectedUser) {
+      this.setState({selectedUser: null});
```

Comment:
Since we're settings state dependent on the previous value, we probably want to use the callback form

```
this.setState(prevState => {
  if (prevState.selectedUser) {
    return { selectedUser: null }
  }

  return null
}
```